browser/components/smartwindow/content/smartbar.mjs

```diff
@@ -255,12 +283,43 @@ export function attachToElement(element, options = {}) {
       ? getQueryTypeIcon(suggestion.type)
       : "🔍";
 
-    const text = document.createElement("span");
-    text.className = "suggestion-text";
-    text.textContent = suggestion.text;
+    const textContainer = document.createElement("div");
+    textContainer.className = "suggestion-text-container";
+
```

Comment:
only feed back and not really even needed but could make this a create element to create a new lit component and encapsulate this a  little if you didn't want to set the classes and just set props instead of the text contents  etc...but this is fine.

Reply:
That is great feedback and I was going back and forth on creating a component for this! I’ll try to opt for more encapsulation with any new items I’m adding. I’ll also come back to this one as soon as we are making less frequent changes to the UX requirements.